Makefile

```diff
@@ -185,5 +185,8 @@ $(eval $(call build-package,fontconfig,2.14.0-r0))
 $(eval $(call build-package,openjdk,17.0.4.1-r0))
 $(eval $(call build-package,su-exec,0.2-r0))
 $(eval $(call build-package,postgresql,11.17-r0))
+$(eval $(call build-package,ko,0.12.0-r0))
+$(eval $(call build-package,apko,0.5.0-r0))
+$(eval $(call build-package,melange,0.1.0-r0))
```

Comment:
Is it worth packaging this version of melange, considering it is too old to build Wolfi?

Reply:
I can drop it if you want. I thought it'd be nice to start having it packaged, and we can bump it when we do the next melange release. If you'd prefer to wait until 0.2.0 I can do that too.

Reply:
We can just hold until apko 0.6 and melange 0.2 are shipped, which i might do tomorrow. :)